netci.groovy

```diff
@@ -49,168 +49,50 @@ build.cmd /no-node-reuse /no-deploy-extension /${configuration.toLowerCase()}
-        description('')
-
-        // This opens the set of build steps that will be run.
-        steps {
-            // Build roslyn-project-system repo - we also need to set certain environment variables for building the repo with VS15 toolset.
-            batchFile("""
-echo *** Build Roslyn Project System ***
-rmdir /S /Q %USERPROFILE%\\.nuget\\packages\\microbuild.plugins.swixbuild
-SET VS150COMNTOOLS=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\Common7\\Tools\\
-SET VSSDK150Install=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\v15.0\\VSSDK\\
-SET VSSDKInstall=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\v15.0\\VSSDK\\
-
-build.cmd /no-node-reuse /release /skiptests
-""")
-
-            // Patch all the MSBuild xaml and targets files from the current roslyn-project-system commit into VS install.
-            batchFile("""
-echo *** Patch the MSBuild xaml and targets ***
-SET VS_MSBUILD_MANAGED=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\Managed
-
-mkdir backup
-xcopy /SIY "%VS_MSBUILD_MANAGED%" .\\backup\\Managed
-
-xcopy /SIY .\\src\\Targets\\*.targets "%VS_MSBUILD_MANAGED%"
-xcopy /SIY .\\bin\\Release\\Rules\\*.xaml "%VS_MSBUILD_MANAGED%"
-""")
-
-            // Restore roslyn nuget packages
-            batchFile("""
-echo *** Restore Roslyn ***
-pushd %WORKSPACE%\\roslyn
-Restore.cmd
-""")
-
-            // Build the SDK and install .NET Core Templates.
-            batchFile("""
-echo *** Build the SDK and install .NET Core Templates  ***
-SET VS150COMNTOOLS=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\Common7\\Tools\\
-SET DeveloperCommandPrompt=%VS150COMNTOOLS%\\VsMSBuildCmd.bat
-
-echo  *** Call VsMSBuildCmd.bat
-call "%DeveloperCommandPrompt%" || goto :BuildFailed "VsMSBuildCmd.bat"
-
-SET VSSDK150Install=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\v15.0\\VSSDK\\
-SET VSSDKInstall=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\v15.0\\VSSDK\\
-SET MSBUILDEXE=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\15.0\\Bin\\MSBuild.exe
+    ['Debug', 'Release'].each { configuration ->
 
-pushd %WORKSPACE%\\sdk
-echo *** Build SDK
-call build.cmd -Configuration release -SkipTests || goto :BuildFailed "SDK"
+        def newVsiJobName = Utilities.getFullJobName(project, "windows_integration_${configuration.toLowerCase()}", isPR)
 
-exit /b 0
+        def newVsiJob = job(newVsiJobName) {
+            // This opens the set of build steps that will be run.
+            steps {
+                // Indicates that a batch script should be run with the build string (see above)
+                // Also available is:
+                // shell (for unix scripting)
+                batchFile("""
+echo *** Installing 2.0 CLI ***
```

Comment:
We are installing the latest CLI so that we get the latest SDK and templates? If so, I think we should install the CLI from the rel/1.0.0 branch for now* because that's the one that's shipping in VS. We first need to make sure the integration of bits in shipping VS is good. We could probably have a different queue that installs the 2.0 CLI and tests as well to make sure that 2.0 changes don't break VS.

* - Btw @livarcocc is figuring out which branch of CLI will get inserted to VS. It's rel/1.0.0 but is going to change soon- we should make sure to update this when that happens.